app/validators/index.js

```diff
@@ -59,7 +59,7 @@ export const validatePassword = (values, { t = i18nT() }) => {
         })
       },
       regex: {
-        pattern: '^((?=.*[0-9])(?=.*[A-Za-z])(?=.*[@$!%*#?&]))',
+        pattern: '^((?=.*[0-9])(?=.*[A-Za-z])(?=.*[~!@#$%^&_+=`|(){}:;\'\\/,.?/*#?&£-]))',
```

Comment:
looks like `/` was added to the pattern twice, may not be need?
The XML output here includes there characters but they are encoded `<>` characters as: `&lt;&gt;`:
https://pages.github.rackspace.com/IX/internal-docs-signup/api-docs/api-reference-v2/ops-validation-v2.html#password-validation

also  `[]` character are in the list of special chars.

```suggestion
        pattern: '^((?=.*[0-9])(?=.*[A-Za-z])(?=.*[~!@#$%^&_+=`|()[\]{}:;\'\\,.?/*#?&<>£-]))',
```